enzyme/Enzyme/ActivityAnalysis.cpp

```diff
@@ -810,6 +810,11 @@ bool ActivityAnalyzer::isConstantInstruction(TypeResults const &TR,
         llvm::errs() << "known inactive intrinsic " << *I << "\n";
       InsertConstantInstruction(TR, I);
       return true;
+    } else if (isIntelSubscriptIntrinsic(*II)) {
+      // The intrinsic "llvm.intel.subscript" does not propogate deriviative
+      // information directly. But its returned pointer may be active.
+      InsertConstantInstruction(TR, I);
+      return true;
```

Comment:
if readnone and returning a pointer, this shouldn't need explicitly.

constant instr, active value [if active source].